openshift/helper/__init__.py

```diff
@@ -8,19 +11,46 @@
 from openshift import client
 from openshift.client.models import V1DeleteOptions
 
-#: Regex for finding versions
+# Regex for finding versions
 VERSION_RX = re.compile("V\d((alpha|beta)\d)?")
 
+# Expected metaclass object name
+METACLASS_NAME = 'V1ObjectMeta'
+BASE_API_VERSION = 'V1'
+
+class OpenShiftException(Exception):
+    """
+    Raised when there is an error inside of an Ansible module.
+    """
+
+    def __init__(self, message, **kwargs):
+        """
+        Creates an instance of the AnsibleModuleError class.
+        :param message: The friendly error message.
+        :type message: basestring
+        :param kwargs: All other keyword arguments.
+        :type kwargs: dict
+        """
+        self.message = message
+        self.value = kwargs
+        self.value['message'] = message
+
+    def __str__(self):
+        """
+        String representation of the instance.
+        """
+        return json.dumps(self.value)
+
 
 class KubernetesObjectHelper(object):
     # TODO: add support for check mode to helper
-    def __init__(self, api_version, kind):
+    def __init__(self, api_version, kind, namespaced=False):
```

Comment:
I think we should avoid adding a namespaced attribute here. I think it would be better to return whether the resource is namespaced as part of the returned argspec data.